wikipedia-sentences/README.md

```diff
@@ -51,7 +60,7 @@ shuf input.txt > input.txt
 ```
 
 To shuffle a file on macos, please read [this post](https://apple.stackexchange.com/questions/142860/install-shuf-on-os-x/195387).
-
+<!---
```

Comment:
What is this? if not necessary, you can delete what is commented?

Reply:
Sure